### ISSUE TITLE
Background Image on top

### ISSUE THREAD

**Issue by pedakas** [reuterbal/photobooth#189]
Hello,
Is it possible in the picture creation process that the background image is on top, so that i can add flowers or something else like in this picture for example: https://sdeweddings.com/torontoweddingvideophoto/2013/09/sdeweddings-toronto-photo-booth-rental-9.jpg

Thanks!

**Comment by andyboeh** [comment]
I suppose #176 is what you are looking for.

**Comment by pedakas** [comment]
> I suppose #176 is what you are looking for.

Thanks, so if i unterstand it correct i just replace this 3 files with the files from patkan and i should be able to add a "watermark" in the settings menu?

**Comment by patkan** [comment]
Yep, that should be it. Alternatively you could use directly the branch at https://github.com/patkan/photobooth/tree/feature-assemble-overlay
In case that you find any issues with it, it would be great, if you could post them in #176

**Comment by pedakas** [comment]
Thanks, I will try this next weekend and let you know if it worked
